### PR TITLE
added better 404 error page

### PR DESCRIPTION
**Pull request by FatherWolf**
made error page and added it where it was needed still need to change the icon.

### CODE REVIEW

**Comment by FatherWolf** [comment]
Screenshot from 2022-08-01 11-07-12

File: src/i18n/resources/en.ts [hunk 1]
```diff
@@ -130,10 +130,12 @@ export const en: LanguageResource = {
     nodeaddress: 'Node Address',
   },
   error: {
-    notfound: 'Not found',
+    notfound:
+      'Looks like you have lost your way, Netmaker has a route for that!',
     network: 'Could not reach server',
     tokenexpire: 'Session has expired',
     unauthorized: 'Unauthorized access attempted',
+    error: '404 Error',
```
Comment (0xdcarns) [inline src/i18n/resources/en.ts]: this should be error404, errorNotFound, or something more descriptive

File: src/i18n/resources/en.ts [hunk 2]
```diff
@@ -143,6 +145,7 @@ export const en: LanguageResource = {
     viewing: 'Viewing Ext Client',
     none: 'No Ext Clients Found',
     name: 'Name',
+
```
Comment (0xdcarns) [inline src/i18n/resources/en.ts]: random blank line

**Review by 0xdcarns — CHANGES_REQUESTED** [review]

File: src/i18n/resources/en.ts (hunk 1 above)
Comment (FatherWolf) [reply src/i18n/resources/en.ts]: going to remove it it isnt needed now must have missed it

**Review by 0xdcarns — APPROVED** [review]

**Pull request merged**